### DISCUSSION TITLE
Using deta with Pipenv instead of requirements.txt

### DISCUSSION THREAD

**Discussion by mrpbennett** [deta/cloud-docs#365] [Help] [answered]
Hey Gang.

It seems to use deta currently you have to create a requirements.txt file. Will this always be the case? I prefer using pipenv in my work flow, and was hoping to use deta with some projects at somepoint.

**Comment by abdelhai** [comment, marked as answer]
hi, soon we will give you the freedom to use your favorite deps management tool. for now, we need a requirements.txt file

**Reply by mrpbennett** [reply]
Great thanks, is there a timeline for this?

**Comment by abdelhai** [comment]
a month or so. lmk if you would like to get early access

**Reply by mrpbennett** [reply]
Nice...that would be great if you could add me :)
